manifest.json

```diff
@@ -51,11 +51,24 @@
 				},
 				"default": "signalbot"
 			},
+			{
+				"name": "bot_synapse_adm",
+				"type": "boolean",
+				"ask": {
+					"en": "Give the WhatsApp bot administrator rights to the Synapse instance?",
+					"fr": "Donner au robot WhatsApp des droits administrateur à l'instance Synapse ?"
+				},
+				"help": {
+					"en": "If true, the bot can group WhatsApp chats in a Matrix space. Not required if you set up Synapse so that non-admins are authorized to create spaces.",
+					"fr": "Si true, le robot groupera les conversations WhatsApp dans un espace Matrix. Pas nécessaire si vous avez réglé Synapse pour qu'il autorise les non-admin à créer des espaces."
+				},
+				"default": true
+			},			
 			{
 				"name": "encryption",
 				"type": "boolean",
 				"ask": {
-					"en": "Enable end-to-bridge encryption ?",
+					"en": "Enable end-to-bridge encryption?",
 					"fr": "Activer le chiffrement entre synapse et le bridge ?"
```

Comment:
end-to-bridge
```suggestion
					"en": "Enable end-to-bridge encryption?",
					"fr": "Activer le chiffrement entre Signal et le bridge ?"
```

Reply:
Right, I meant the signal app on a user device.

Signal app --(encrypted)--> signald ==(unencrypted)==> mautrix_signal --(encrypted)--(synapse)--> matrix user device

Reply:
we could explain this in the README.